cluster/defaults.go

```diff
@@ -323,6 +323,15 @@ func (c *Cluster) setClusterServicesDefaults() {
 		}
 
 	}
+
+	enableEncryptionByDefault, err := checkVersionNeedsEncryptionDefault(c.Version)
+	if err != nil {
+		logrus.Warnf("Cannot determine if cluster version [%s] needs to have encryption enabled by default: %v", c.Version, err)
+	}
+	if enableEncryptionByDefault && c.Services.KubeAPI.SecretsEncryptionConfig == nil {
+		logrus.Debugf("Enabling encryption of secret data at rest by default for cluster version [%s]", c.Version)
+		c.Services.KubeAPI.SecretsEncryptionConfig = newDefaultSecretsEncryptionConfig()
```

Comment:
nit: can this be:
```suggestion
		c.Services.KubeAPI.SecretsEncryptionConfig = &v3.SecretsEncryptionConfig{
		Enabled: true,
	}
```
rest lgtm

Reply:
Updated in the latest commit!